packages/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": true,
+  "printWidth": 120
```

Comment:
This file prevent different prettier format on packages. We can remove this later